RecoLocalCalo/HcalRecAlgos/interface/MahiFit.h

```diff
@@ -127,7 +127,7 @@ class MahiFit
 
   void solveSubmatrix(PulseMatrix& mat, PulseVector& invec, PulseVector& outvec, unsigned nP) const;
 
-  double getSiPMDarkCurrent(double darkCurrent, double fcByPE, double lambda) const;
+  inline double getSiPMDarkCurrent(double darkCurrent, double fcByPE, double lambda) const { double mu = darkCurrent * 25 / fcByPE; return sqrt(mu/pow(1-lambda,3)) * fcByPE;}
```

Comment:
please add a linebreak for readability

Reply:
```return sqrt(darkCurrent*fcByPE/pow(1-lambda,3))*5``` 
should give the same result and has one less division.
the bit-wise results may change though and this optimization is perhaps not so essential. So, feel free to ignore.

Reply:
in any case, this function isn't really used anymore, since we have the effective pedestals in the database now